tools/make_patch.c

```diff
@@ -0,0 +1,532 @@
+#define PROGRAM_NAME "make_patch"
+#define USAGE_OPTS "labels.sym constants.txt patched.gbc original.gbc vc.patch.template vc.patch"
```

Comment:
To match the actual constants file extension:
```suggestion
#define USAGE_OPTS "labels.sym constants.out patched.gbc original.gbc vc.patch.template vc.patch"
```